molecule/default/molecule.yml

```diff
@@ -14,14 +14,14 @@ platforms:
     image: docker.io/linuxsystemroles/centos-6
     privileged: true
   - name: centos-7
-    image: centos/systemd
+    image: registry.centos.org/centos/systemd
```

Comment:
IMHO this should be amended in the first commit to simplify the review.

Reply:
> IMHO this should be amended in the first commit to simplify the review.

. . . while I thought it would make it easier to review as a followup commit, then squash into a single commit later - but if you would rather have all of the changes made to address review comments in a single commit, ok . . .

Reply:
If you intend to squash the commits, using a fixup commit might be a good idea to indicate this. If the commits are grouped together, they could be easily selected in the commit selection box when reviewing.

In practice I rareley need to see the diff between the original commit and the followup commit since I review the commit by itself. If they are concise  and contain only one change, it is also easy enough to always review them in full.

Reply:
I broke up the commits for this PR into discrete but logical units.